stunnel/stunnel_cache.mli

```diff
@@ -26,8 +26,8 @@
     will be used, otherwise we make a fresh one. *)
 val with_connect :
   ?use_fork_exec_helper:bool ->
-  ?write_to_log:(string -> unit) -> string -> int -> bool ->
-   (Stunnel.t -> 'b) -> 'b
+  ?write_to_log:(string -> unit) ->
+  ?verify_cert:bool -> string -> int -> (Stunnel.t -> 'b) -> 'b
```

Comment:
I understood that `verify_cert` needs to be `bool option`:
* None: use global default
* Some value: use that value

Reply:
It is a bool option in disguise (this is implicit with the `?` syntactic sugar)

Reply:
See stunnel/stunnel.ml +273
```
let must_verify_cert verify_cert =
  match verify_cert with
  | Some x -> x
  | None -> Sys.file_exists verify_certificates_ctrl
```